package.json

```diff
@@ -24,20 +24,20 @@
     "typeorm-koa-example"
   ],
   "devDependencies": {
-    "typescript": "^2.1.5"
+    "typescript": "^2.4.2"
   },
   "dependencies": {
     "@types/koa": "^2.0.39",
-    "@types/koa-bodyparser": "^3.0.23",
-    "@types/koa-router": "^7.0.21",
-    "@types/node": "^7.0.4",
-    "body-parser": "^1.17.1",
-    "koa": "^2.2.0",
+    "@types/koa-bodyparser": "^3.0.24",
+    "@types/koa-router": "^7.0.23",
+    "@types/node": "^8.0.24",
+    "body-parser": "^1.17.2",
+    "koa": "^2.3.0",
     "koa-bodyparser": "^3.2.0",
```

Comment:
This could be upgraded to `^4.2.0`, assuming there is a warning that a non LTS version of Node is needed (`>=7.6.0`)